README.md

```diff
@@ -27,7 +27,7 @@ The following Environment Variables can be defined:
 ```
 #!/usr/bin/env sh
 
-export SERVICE_ENTITY_ID=... # The entity id of the service using Verify Service Provider
+export SERVICE_ENTITY_ID=... # The entity id of the service using Verify Service Provider. This will need to be in quotes.
```

Comment:
Not necessarily clear from here that the _list_ needs to be in single quotes AND that they should really be double quote escaping each value (bash best practice). Nor is it particularly obvious that it is just a space separated list (although that is a fairly bash-y thing to do). Is there a reason we went for space separated as opposed to a JSON ["x", "y"] list?

Reply:
It shouldn't be space separated?